debian/libnoopenh264-6.links

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/dh-exec
+[linux-amd64] /var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/x86_64/2.0/active/files/extra/libopenh264.so.6 /usr/lib/${DEB_HOST_MULTIARCH}/openh264/extra/libopenh264.so.6
+[linux-arm64] /var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/aarch64/2.0/active/files/extra/libopenh264.so.6 /usr/lib/${DEB_HOST_MULTIARCH}/openh264/extra/libopenh264.so.6
+[linux-arm linux-armel linux-armhf] /var/lib/flatpak/runtime/org.freedesktop.Platform.openh264/arm/2.0/active/files/extra/libopenh264.so.6 /usr/lib/${DEB_HOST_MULTIARCH}/openh264/extra/libopenh264.so.6
```

Comment:
Let's drop this hunk – we don't support armhf any more.

Reply:
Ok, but does it hurt? I can remove but just wondering.

Reply:
I guess it's harmless